packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -621,7 +621,7 @@ class Viewport implements IViewport {
    *   be detected for pan/zoom values)
    * @returns boolean
    */
-  protected resetCamera(
+  public resetCamera(
```

Comment:
Why are you making this public, protected is good enough

Reply:
Hi @sedghi, resetCamera should be public in Viewport.ts because in the others viewports classes it convert it to public 
VolumeViewport.ts:
``  resetCamera(
    resetPan?: boolean,
    resetZoom?: boolean,
    resetToCenter?: boolean
  ): boolean {
     return super.resetCamera(resetPan, resetZoom, resetToCenter);
  }
``
VolumeViewport3D.ts:
``  public resetCamera(
    resetPan = true,
    resetZoom = true,
    resetToCenter = true
  ): boolean {
    super.resetCamera(resetPan, resetZoom, resetToCenter);
    this.resetVolumeViewportClippingRange();
    return;
  }``